src/common.ts

```diff
@@ -18,6 +18,13 @@ export function isConfig(arg: Rule | Config): arg is Config {
   return (arg as Config).config !== undefined;
 }
 
+export function isSubCollection(collection: string): boolean {
+  const filteredCollections = collection.split('/').filter(c => c.length > 0);
+  const numCollections = filteredCollections.length;
+  // Return false is equal to 1 or an even number
+  return numCollections !== 1 && numCollections % 2 === 1;
+}
```

Comment:
not sure how this logic means it's a subcollection? But more to the point do we need to restrict it to subcollections now? As we have an explicit flag we could allow it to work on any collection?

Reply:
Not really. The explicit flag should be enough